todo_app/templates/index.html

```diff
@@ -11,8 +11,21 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
+        {% for item in items %}
+          <li>{{ item.title }}</li>
+        {% endfor %}
         <!-- List the todo items here -->
       </ul>
     </div>
   </div>
+
+  <form action="/newitem" method="POST" class="form_newitem">
```

Comment:
Slightly odd use of `class`. This `form` is a distinctly different type of element from the `div` elements below, and actually contains them. So it's surprising that you'd ever want the same class on them.

But also the class isn't doing anything at the moment, so you could just get rid of all 3.